.github/workflows/test.yml

```diff
@@ -33,15 +33,15 @@ jobs:
       max-parallel: 1 # tests interfere with each other if run in parallel
       matrix:
         # order matters - this setup minimizes the interference between jobs
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9]
```

Comment:
python 2.7 is no longer supported https://github.com/actions/setup-python/issues/672
3.5, 3.6, and 3.7 only use more specific version like x.y.z and do not have mac arm64 versions
proposing to simply delete them but happy to find an alternative based on preferences